pkg/apis/autoscaling/register.go

```diff
@@ -239,7 +239,7 @@ var (
 	}
 	MinScaleAnnotation = kmap.KeyPriority{
 		MinScaleAnnotationKey,
-		GroupName + "/minScale",
```

Comment:
These legacy keys should not be removed - there are older clients/documentation that has `/minScale` and we should preserve those

Reply:
In the documentation, it would be good to have the older versions documented somewhere but listed as deprecated, since not everyone has the luxury of upgrading to latest. 

The change to kebab case was not included in the release notes and it's not easy to determine which annotations were changed in the source code and at what version.

Reply:
@xtaje would you mind opening an issue in the https://github.com/knative/docs/ repo for this?

Reply:
Added in https://github.com/knative/docs/issues/5090

Reply:
@dprotaso in this line 263 below, I am retaining the `/min-scale` annotation key using the `DeprecatedMinScaleAnnotationKey` variable